### PR TITLE
[INLONG-2187] support audit view

### LINKED ISSUE #2187: [Feature] Website support audit view
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by leezng**
### Title Name: [INLONG-XYZ][component] Title of the pull request

where *XYZ* should be replaced by the actual issue number.

Fixes #2187

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by dockerzhang — APPROVED** [review]

File: inlong-dashboard/src/pages/AccessDetail/Audit/config.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+ * under the License.
+ */
+
+import React from 'react';
+import { Button } from 'antd';
+import dayjs from 'dayjs';
+
+export const auditList = [
+  {
+    label: 'InLongApi接收成功',
```
Comment (gosonzhang) [inline inlong-dashboard/src/pages/AccessDetail/Audit/config.tsx:26]: Is the front-end already support multiple languages?
if so, this information should be configured and turned into multiple languages

Comment (leezng) [reply inlong-dashboard/src/pages/AccessDetail/Audit/config.tsx:26]: I18n will add at another PR.

Comment (gosonzhang) [reply inlong-dashboard/src/pages/AccessDetail/Audit/config.tsx:26]: ok

**Review by gosonzhang — APPROVED** [review]
+1

**Pull request merged**